globals/plotTypes.json

```diff
@@ -0,0 +1,83 @@
+{
+  "m": {
+    "x": [
+      "m",
+      "sigma",
+      "radii",
+      "lnsigma",
+      "halo_bias",
+      "n_eff",
+      "dndm",
+      "ngtm",
+      "dndlnm",
+      "dndlog10m",
+      "rho_gtm",
+      "rho_ltm"
+    ],
+    "y": [
+      "central_occupation",
+      "cmz_relation",
+      "dndlnm",
+      "dndlog10m",
+      "dndm",
+      "fsigma",
+      "halo_bias",
+      "how_big",
+      "lnsigma",
+      "n_eff",
+      "ngtm",
+      "radii",
+      "rho_gtm",
+      "rho_ltm",
+      "satellite_occupation",
+      "sigma",
+      "total_occupation",
+      "tracer_cmz_relation",
+      "k_hm"
+    ]
```

Comment:
How did you decide what should / shouldn't be in x vs y?

Reply:
I didn't. It's Steven's list.